src/components/Login/Login.jsx

```diff
@@ -18,23 +22,43 @@ const Login = () => {
   return (
     <>
       <AppHeader />
-      <section id="login" className="panel">
-        <div className="row">
-          <label id="labelLogin" htmlFor="btnLogin">
-            Click the following login button to log into your pod at [
-            <a href={SOLID_IDENTITY_PROVIDER} target="_blank" rel="noreferrer">
-              {SOLID_IDENTITY_PROVIDER}
-            </a>
-            ]:{' '}
-          </label>
-          <LoginButton
-            oidcIssuer={SOLID_IDENTITY_PROVIDER}
-            redirectUrl={redirectUrl}
-            onError={console.error}
-          />
-          <p>Not logged in</p>
-        </div>
-      </section>
+      <Container component="main" maxWidth="xs">
+        <Box
+          sx={{
+            marginTop: 5,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            textAlign: 'center',
+            padding: '20px'
+          }}
+        >
+          <Paper elevation={2} sx={{ display: 'inline-block', mx: '2px', padding: '20px' }}>
+            <section id="login">
+              <div className="row">
+                <label id="labelLogin" htmlFor="btnLogin">
```

Comment:
Thanks @xscottxbrownx. I did miss those.

For Line 39, htmlFor was for when previously used when we're still using Inrupt's Solid JS client library for a customized button instead of Inrupt's Solid SDK's LoginButton component. We're no longer using that and it seems like we haven't removed it from before.

We should just remove the htmlFor="btnLogin". It no longer used in Login.jsx.